wls-gui-admintool/src/composables/common/apiUtils.ts

```diff
@@ -0,0 +1,17 @@
+import type { AxiosResponse } from "axios";
+
+export function useApiUtils() {
+  function returnUndefinedOnStatus204OrElseDate<T>(
```

Comment:
würde es nicht auf `Date` beziehen, damit man die funktion auch für andere Services verwenden kann
```suggestion
  function returnUndefinedOnStatus204OrElseResponsedata<T>(
```

Reply:
ist ein typo^^. Wollte Data verwenden.

Reply:
refaktored mit a0b8b6df0b677232e3c3a0598bac5900764f95ab